CONTRIBUTING.md

```diff
@@ -107,3 +107,7 @@ Then run `test/run_tests.sh` and `test/cpp/run_tests.sh` to verify the setup is
 ### Useful materials
 1. [OP Lowering Guide](https://github.com/pytorch/xla/blob/master/OP_LOWERING_GUIDE.md)
 2. [CODEGEN MIGRATION GUIDE](https://github.com/pytorch/xla/blob/master/CODEGEN_MIGRATION_GUIDE.md)
+
+### Sharp Edges
+
+* If local changes aren't visible, ensure you built both PyTorch and PyTorch/XLA with `python setup.py develop` and not `python setup.py install`. If you're unsure, uninstall existing pytorch/xla with `pip uninstall torch_xla` and `pip uninstall torch`, then rebuild PyTorch and PyTorch/XLA.
```

Comment:
I think the issue you ran into was using `develop` and then `install` (or the opposite) which can create problems. I generally `install` PyTorch and `develop` PyTorch/XLA without issue. I would shorten the advice here to:

>  If local changes aren't visible, uninstall existing pytorch/xla with `pip uninstall torch_xla` and `pip uninstall torch`, then rebuild PyTorch and PyTorch/XLA with `python setup.py develop` or `python setup.py install`

Reply:
Thanks updated the text to reflect what you have here.